### PR TITLE
chore(docs): remove extra punctuation and duplicate example

### PR DESCRIPTION
**Pull request by robin-norwood**
The examples for the `convert` command included an extra "\`", and one of the examples is a duplicate.

### CODE REVIEW

**Review by ansgarm — APPROVED** [review]

File: website/docs/cdktf/cli-reference/commands.mdx [hunk 1]
```diff
@@ -109,13 +109,7 @@ Examples:
 Convert a local file.
 
 ```bash
-cat main.tf | cdktf convert > imported.ts`
-```
-
-Convert HCL in your clipboard to Python on OSX.
-
-```bash
-pbpaste | cdktf convert --language python | pbcopy
```
Comment (DanielMSchmidt) [inline website/docs/cdktf/cli-reference/commands.mdx:118]: It has an extra --language python attribute though 🤔  and also pastes into pbcopy

**Comment by laurapacilio** [comment]
I defer to the team on this :)

File: website/docs/cdktf/cli-reference/commands.mdx (hunk 1 above)
Comment (robin-norwood) [reply website/docs/cdktf/cli-reference/commands.mdx:118]: It's the next example (line 124ish) that's identical to this one.

Comment (DanielMSchmidt) [reply website/docs/cdktf/cli-reference/commands.mdx:118]: Ah I see 👍

**Review by DanielMSchmidt — APPROVED** [review]

**Pull request merged**